lib/src/Base/Func/P1LagrangeInterpolation.cxx

```diff
@@ -207,18 +190,17 @@ void P1LagrangeInterpolation::computeProjection()
   }
 }
 
-Field P1LagrangeInterpolation::operator()(const Field & field) const
+Sample P1LagrangeInterpolation::operator()(const Sample & values) const
```

Comment:
@jschueller My bad, I forgot to increment callsNumber_ in this method, can you please fix it?

Reply:
done